api/src/modules/sourcing-locations/sourcing-locations.controller.ts

```diff
@@ -38,6 +38,7 @@ import { UpdateSourcingLocationDto } from 'modules/sourcing-locations/dto/update
 import { PaginationMeta } from 'utils/app-base.service';
 import { SourcingLocationsMaterialsResponseDto } from 'modules/sourcing-locations/dto/materials.sourcing-location.dto';
 import { SourcingLocationsMaterialsSerializer } from 'modules/sourcing-locations/serializer/sourcing-locations-materials.serializer';
+import { GetSourcingMaterialsQueryDto } from './dto/materials-query.sourcing-location.dto';
```

Comment:
this relative import should cause ESLint to report a problem. Please check your IDE settings, and use an absolute import instead